legend-engine-xt-serviceStore-grammar/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/IServiceStoreCompilerExtension.java

```diff
@@ -31,7 +30,7 @@ static List<IServiceStoreCompilerExtension> getExtensions()
         return Lists.mutable.withAll(ServiceLoader.load(IServiceStoreCompilerExtension.class));
     }
 
-    default List<Function<SecurityScheme, Root_meta_external_store_service_metamodel_SecurityScheme>> getExtraSecuritySchemeProcessors()
+    default List<Function2<String, CompileContext, Root_meta_external_store_service_metamodel_SecuritySchemeRequirement>> getExtraSecuritySchemeProcessors()
```

Comment:
lets have earlier function marked as deprecated for backward compatibility we will remove in next commit
Lets add a new function